### DISCUSSION TITLE
Include Rule IDs in ruff-format Output for Better Insight into Changes

### DISCUSSION THREAD

**Discussion by zbambusek** [astral-sh/ruff#14278] [Ideas]
When the following message is displayed,
```
ruff-format..............................................................Failed
hook id: ruff-format
- files were modified by this hook
1 file reformatted

```
it would be very helpful if it also included the IDs of the specific rules that were applied during reformatting. This would allow users to better understand the changes made by ruff and make it easier to adjust configurations as needed.

**Comment by MichaReiser** [comment]
Thanks for the suggestion. 

Ruff's formatter doesn't use individual rules. Instead, it holisticly applies an entire code style. That's why we can't expose such a rule id
